en/release-notes/upgrade-guide-v2.1.md

```diff
@@ -0,0 +1,164 @@
+# Cocos Creator v2.1.0 Release Notes
+
+__Cocos Creator__ __v2.1__, adds support for 3D features such as: **3D model rendering**, **3D Camera**, **3D skeletal animation**, and **3D point selection**. The __Cocos Creator__ editor now natively supports parsing 3D model files in **FBX format**. It's that simple, no additional workflow is required!
+
+In the previous version, v2.0, __Cocos Creator__ had used a __Cocos3D__ based 3D renderer to officially introduce 3D support. The addition of 3D features can greatly enrich the expressiveness of 2D games, reduce the resource overhead of 2D games, and bring more imagination and ideas to expand games.
+However, it should be emphasized that the target users of __Cocos Creator__ is still 2D game development, and the **2D game is still the core** service of __Cocos Creator 2.x__. We will continue to optimize the development experience and basic performance of 2D games, and gradually increase the 3D functions suitable for integration into 2D games. Therefore, __Cocos Creator__ does not allow a pure 3D game, which is the goal of __Cocos3D__. __Cocos Creator's__ development path is closer to 2.5D.
+
+This is not the first time __Cocos__ has upgraded the engine from 2D to 3D. A few years ago, __Cocos2d-x__ expanded to the 3D space and accumulated a lot of experience. It also produced a very good game concept like *Sword and Home*. However, at that time, 3D functions were piled up on a very heavy 2D renderer, and there was no editor support. It was not widely recognized by the market. This time, __Cocos Creator__ is based on __Cocos3D's__ 3D renderer, which has a lot of 3D stability and guarantees a seamless editor experience.
```

Comment:
1. It also produced a very good game concept like *Sword and Home*.  I think we can get rid of the "concept"
2. *Sword and Home* -> 《Art of Conquest》
3. I think we can divide sentences like this: and there was no editor support, so it was...

Reply:
1. fixed

2. fixed

3. The sentence is fine, technically, but I divided it out for easier reading.